### ISSUE TITLE
System.Web.Mvc amd System.Web.WebPages are specified as frameworkAssembly again (regression)

### ISSUE THREAD

**Issue by reijerh** [RazorGenerator/RazorGenerator#118]
System.Web.Mvc and System.Web.WebPages are nuget package dependencies nowadays, available in Microsoft.AspNet.Mvc and Microsoft.AspNet.WebPages. They were first specified this way in https://github.com/RazorGenerator/RazorGenerator/pull/73 (issue: https://github.com/RazorGenerator/RazorGenerator/issues/5).

However, in the 2.4.7 I'm currently using it seems they are once again frameworkAssembly references:
```
    <dependencies>
      <dependency id="WebActivatorEx" version="2.0.3" />
      <dependency id="Microsoft.AspNet.Mvc" version="3.0.50813.1" />
    </dependencies>
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.0" />
      <frameworkAssembly assemblyName="System.Web.Mvc" targetFramework=".NETFramework4.0" />
      <frameworkAssembly assemblyName="System.Web.WebPages" targetFramework=".NETFramework4.0" />
    </frameworkAssemblies>
```

It seems this was done in https://github.com/RazorGenerator/RazorGenerator/commit/6252eee5a3cace9347935d862262f81648b6bd2e

I don't know if this was done on purpose, but it results in build conflicts between version 3.0.0.1 (GAC) and 5.2.3.0 (package). I think they should be referenced as packages again.

**Comment by CZEMacLeod** [comment]
Yeah - had to manually add code to my deployment system to patch the nuspec file and ignore these as they are not in the GAC and throw an error in nuget restore when adding references.
System.Web is obviously valid, but not the other two.

**Comment by reijerh** [comment]
@CZEMacLeod am I correct in assuming this is being fixed by your open PR?

**Comment by CZEMacLeod** [comment]
Yes - It is included in that PR.
